### DISCUSSION TITLE
vc-drawings 绘制图标点 编辑bug

### DISCUSSION THREAD

**Discussion by heroqyc** [zouyaoji/vue-cesium#599] [Q&A]
vc-drawings 绘制图标点  在可编辑模式下  右键放弃修改图标点位置时 会出现一个 label 提示框无法消失
![QQ截图20240910114802](https://github.com/user-attachments/assets/8ab518da-c92e-476e-b998-e7875301e6bf)

**Comment by zouyaoji** [comment]
https://github.com/zouyaoji/vue-cesium/issues/600
